db/migrate/20150111192335_subscribe_old_changesets.rb

```diff
@@ -1,4 +1,7 @@
 class SubscribeOldChangesets < ActiveRecord::Migration[4.2]
+  class Changeset < ActiveRecord::Base
+  end
+
```

Comment:
I think this will fail if there are changesets in the database as the migration relies on the `subscribers` and `user` associations on the changeset.